// Copyright (c) 2008 AdCopy
// Author: Jeff Weisberg
// Created: 2008-Dec-15 19:19 (EST)
// Function: 
//
// $Id$

message ACPStdReply {
	required int32		status_code	= 1;
	optional string		status_message	= 2;
}

